src/components/smart_objects/include/smart_objects/array_schema_item.h

```diff
@@ -80,13 +80,17 @@ class CArraySchemaItem : public ISchemaItem {
       const utils::SemanticVersion& MessageVersion = utils::SemanticVersion(),
       const bool allow_unknown_enums = false) OVERRIDE;
 
+  bool filterInvalidEnums(SmartObject& Object,
```

Comment:
Shouldn't we start from capital letter?

Reply:
Perhaps by normal case rules, I was following the standards of this class in this case (ex. `validate`, `applySchema`, etc.)